### PR TITLE
feat: upgrade to eslint6. BREAKING CHANGE: new version of airbnb rules

### PR DESCRIPTION
**Pull request by jithine**
## Context

Improve eslint rules

## Objective

pull in new version of base airbnb rule
Rule updates:
   remove conflicting indent rule
   increase line length to 120
   prefer-object-spread to warn

## References 

https://github.com/screwdriver-cd/screwdriver/issues/1650

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
Need to make sure you have `feat(): ... BREAKING CHANGE: ...` in merge title for semantic release to bump major version
https://docs.screwdriver.cd/about/contributing/

**Review by adong — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -28,8 +28,7 @@ module.exports = {
                 peerDependencies: false
             }
         ],
-        indent: ['error', 4],
```
Comment (tkyi) [inline index.js:31]: we're removing indent checking?

Comment (jithine) [reply index.js:31]: it's handled by `prettier` both of them have certain incompatible rules. https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/40/files#diff-168726dbe96b3ce427e7fedce31bb0bcL52

**Review by tkyi — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (adong) [reply index.js:31]: It's handled here.

```
        'prettier/prettier': [
            'error',
            {
                singleQuote: true,
                tabWidth: 4,
                printWidth: 120
            }
        ],
```

https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/40/files#diff-168726dbe96b3ce427e7fedce31bb0bcR65-R71

**Pull request merged**